cloud-storage/provider_common.h

```diff
@@ -0,0 +1,44 @@
+/* Copyright  (C) 2020 The RetroArch team
+ *
+ * ---------------------------------------------------------------------------------------
+ * The following license statement only applies to this file (provider_common.h).
+ * ---------------------------------------------------------------------------------------
+ *
+ * Permission is hereby granted, free of charge,
+ * to any person obtaining a copy of this software and associated documentation files (the "Software"),
+ * to deal in the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software,
+ * and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+ * IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+ * WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#ifndef _CLOUD_STORAGE_PROVIDER_COMMON_H
+#define _CLOUD_STORAGE_PROVIDER_COMMON_H
+
+#include <retro_common_api.h>
+
+RETRO_BEGIN_DECLS
+
+/**
+ * @brief Get the hex encoded MD5 hash of a local file
+ *
+ * Calculates the MD5 checksum of the contents of a local file. The checksum
+ * is hex encoded and returned a newly allocated char*. Caller must release
+ * the returned string.
+ *
+ * @param absolute_filename absolute filename of the local file
+ * @return hex encoded MD5 checksum of file contents
+ */
+char *cloud_storage_get_md5_hash(char *absolute_filename);
```

Comment:
As mentioned before you can move this to common and you can get rid of this file then.

Reply:
This was moved to libretro-common.